WcaOnRails/db/migrate/20180623171213_add_registration_requirements_columns_to_competitions.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddRegistrationRequirementsColumnsToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :free_competition_enabled, :boolean
+    add_column :Competitions, :on_the_spot_registration_enabled, :boolean
+    add_column :Competitions, :on_the_spot_entry_fee, :boolean
+    add_column :Competitions, :on_the_spot_entry_fee_lowest_denomination, :integer, null: false, default: 0
+    add_column :Competitions, :refund_policy, :boolean
```

Comment:
As discussed offline, I don't think we should have a boolean field here. We should always require people to specify the refund policy.

Reply:
This maybe leads to confusion, but the behavior now is: if `renfund_policy = true`, you can select an amount to refund and a limit date, if `renfund_policy = false`, it will be displayed that there won't be refunds under any circumstance, which is a refund policy on itself. Does that make sense?

Reply:
I do think that is confusing. I don't think we need a boolean `refund_policy` to represent all of the things we care about:

- We can represent "50% refund until Monday" with the `refund_policy_amount` and `refund_policy_limit_date` columns you created.
- We can represent "no refunds under any circumstance" with `refund_policy_amount = 0`, and it doesn't even matter what `refund_policy_limit_date` is equal to.